ern-local-cli/src/commands/create-api-impl.js

```diff
@@ -64,8 +62,12 @@ exports.handler = async function ({
   outputDirectory: string,
   hasConfig: boolean
 }) {
+  // react-native-name-api@0.0.3 should be created as react-native-name-api-impl@0.0.3
+  // @org/react-native-name-api@0.0.3 should be created as @org/react-native-name-api-impl@0.0.3
+  // react-native-name-api should be created as react-native-name-api-impl
+  const apiImpl = api.lastIndexOf('@') > 1 ? `${api}`.slice(0, api.lastIndexOf('@')) + '-impl@' + `${api}`.slice(api.lastIndexOf('@') + 1) : `${api}-impl`
```

Comment:
tried https://eslint.org/docs/rules/multiline-ternary
`standardjs` doesn't allow

Reply:
the version number of API or apiImpl may not be same, we should not append the api version to api-impl.

Reply:
Also, it would be best to compose a Dependency object here create impl name out of that ?
Api dependency can be moved up for reuse as below:
```
const apiDep:Dependency = Dependency.fromPath(DependencyPath.fromString(api))
const apiImpl = `${apiDep.name}-impl`

 await generateApiImpl({
      apiDependency: apiDep,
```

Reply:
Another thing we need to keep in mind is that a file path or git path can be passed as an API as well, using a dependency will be ideal in that case.

Reply:
Can you please provide the different arguments that can go with this command? As per slack the conversation `react-native-name-api@0.0.3` -> react-native-name-api-impl@0.0.3

Reply:
argument -> react-native-name-api@0.0.3 , prompt `react-native-name-api-impl` ??

Reply:
w.r.t `file path or git path `, wouldn't the `create-api` , `create-miniapp` get impacted too? Please elaborate. Thanks

Reply:
` react-native-name-api@0.0.3 , prompt react-native-name-api-impl ` Yes

It will not impact `create-api , create-miniapp` since they are not dependent on anything else, whereas an api-impl is dependent on an api. 

you could do the following,

`ern create-api-impl git@github.com:electrode-io/electrode-native.git`
`ern create-api-impl file://path`

This may not be supported currently but will be added in the future. 
https://github.com/electrode-io/electrode-native/blob/master/ern-util/src/Dependency.js#L52